subprojects/launcher/launcher.gradle

```diff
@@ -12,11 +12,11 @@ dependencies {
     compile project(':jvmServices')
     compile project(':core')
     compile project(':cli')
-    compile project(':ui')
     compile project(':toolingApi')
     compile project(':native')
     compile project(':logging')
 
+    compile libraries.commons_io
```

Comment:
`libraries.commons_io` was coming as [a transitive dependency of `compile project(':ui')`](https://github.com/gradle/gradle/pull/1749/files#diff-d527a28ac411deca36c287fe8b663190L26). To allow `:launcher` project to compile, this was required. We can look at the code to remove this dependency if you think it shouldn't be required.